binder/requirements.txt

```diff
@@ -0,0 +1,6 @@
+numpy>=1.19.4; python_version<"3.7"
+numpy>=1.20.0; python_version>="3.7"
+scipy>=1.5.2
+six>1.13.0
+mfem
```

Comment:
Question about this; when you list a requirement here it means you're pulling it from PyPI but `mfem` is _this_ repository so instead of building the current head of this repository you're getting the latest prebuilt version from PyPI? So, for example, adding a new feature, updating a notebook, but not releasing to PyPI would be the notebook wouldn't work as expected?